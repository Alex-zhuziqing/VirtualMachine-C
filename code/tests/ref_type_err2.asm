FUNC LABEL 0
    MOV STK A VAL 9
    REF VAL 8 STK A
    RET
