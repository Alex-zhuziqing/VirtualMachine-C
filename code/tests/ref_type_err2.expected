Cannot assign the address to the type VAL.
